example_configs/toy_authentication.yml

```diff
@@ -30,3 +30,4 @@ access_control:
 trees:
   - path: /
     tree: tiled.examples.toy_authentication:tree
+prefix: /a/b/c
```

Comment:
What do you think of `path_prefex` to make it a little clearer?

Reply:
<butting in here with unsolicited opinion...>  :-)

Aren't there several "paths" involved also?  The public path (URL) the backend path where the data resides (file path), and maybe more?

Maybe `url_path_prefix` or `url_prefix` or `http_root_path` helps distinguish without being too long?

Reply:
Yeah, agreed `prefix` is too overloaded to stand. Will change to one of these.